db/db.go

```diff
@@ -152,6 +158,18 @@ func InitDB(conn *pgxpool.Conn) error {
 	if err != nil {
 		return err
 	}
+	ctx = context.Background()
+	_, err = conn.Exec(ctx, `CREATE INDEX IF NOT EXISTS idx_content_metadata ON txs USING GIN(jsonb_to_tsvector('english', tx #> '{"tx", "body", "messages", 0, "record", "contentMetadata"}' , '["string"]'))`)
+	if err != nil {
+		return err
+	}
+	_, err = conn.Exec(ctx, `CREATE INDEX IF NOT EXISTS idx_record ON txs USING GIN((tx #> '{tx, body, messages, 0, record}') jsonb_path_ops)`)
+	if err != nil {
+		return err
+	}
+	if _, err = conn.Exec(ctx, `CREATE INDEX IF NOT EXISTS idx_keywords ON txs USING GIN((string_to_array(tx #>> '{tx, body, messages, 0, record, contentMetadata, keywords}', ',')))`); err != nil {
+		return err
+	}
```

Comment:
Is it possible to tell Postgres to index all messages instead of only the index `0`?

Reply:
I tried to update the query by using `jsonb_array_elements`. The query works, but it seems that set-returning function cannot be indexed. So the query is very slow.
```
set enable_indexscan = off;
create index if not exists idx_records on txs using GIN((jsonb_array_elements(txs.tx #> '{tx, body, messages}')));

explain select id, record
from txs, jsonb_array_elements(txs.tx #> '{tx, body, messages}') as record
where record -> 'record' @> '{"contentFingerprints": ["$FOOTPRINT"]}'
order by id desc
limit 10;
```

Reply:
I think we can mark a TODO comment and stick with [0], since we don't really use multiple createISCN call in a tx for now
It will be a problem in the future though